var/spack/repos/builtin/packages/libglvnd-fe/package.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class LibglvndFe(BundlePackage):
+    """The GL Vendor-Neutral Dispatch library (Frontend Dummy Package)
+
+    libglvnd is a vendor-neutral dispatch layer for arbitrating OpenGL API
+    calls between multiple vendors. It allows multiple drivers from different
+    vendors to coexist on the same filesystem, and determines which vendor to
+    dispatch each API call to at runtime.
+
+    Both GLX and EGL are supported, in any combination with OpenGL and OpenGL
+    ES."""
+
+    homepage = "https://github.com/NVIDIA/libglvnd"
+
+    version('1.1.1', sha256='71918ed1261e4eece18c0b74b50dc62c0237b8d526e83277ef078554544720b9')
```

Comment:
Bump this to current 1.3.2, same with the base `libglvnd` package.

Reply:
It looks like `v1.2.0` is the "latest" release?  Github seems to be treating the three more recent tags differently and Spack can't download them using the existing URL.